zkchannels-contract/zkchannel_smartpy_script.py

```diff
@@ -34,34 +53,42 @@ class ZkChannel(sp.Contract):
+                  merchBal          = merchFunding,     # merchant's balance. The contract is initialized with this value set to the merchant's initial balance. If the channel is funded by the customer alone, merchBal is set to 0 and addMerchFunding is not called. If it is a dual funded channel, the merchant's initial balance must be funded in one operation using the addMerchFunding entrypoint.  
+                  status            = sp.nat(AWAITING_CUST_FUNDING), # contract status. See above 
+                    # for all the possible zkChannel contract statuses.
+                  revLock           = sp.bytes("0x00"), # revLock initialized to 0x00. When the 
+                    # customer calls custClose, the revocation lock will be passed in as an argument
+                    # and stored in revLock. If the merchant has the revocation secret corresponding
+                    # to revLock, they can claim the entire balance using the merchDispute entrypoint.
                   selfDelay         = selfDelay,        # An enforced delay period that must have
-                  # elapsed between calling custClose and custClaim, and between calling expiry and merchClaim.
-                  delayExpiry       = sp.timestamp(0),  # if the delay is triggered, delayExpiry records when the delay is due to expire
-                  g2                = g2,               # PS pubkey
-                  y2s_0             = y2s_0,            # PS pubkey
-                  y2s_1             = y2s_1,            # PS pubkey
-                  y2s_2             = y2s_2,            # PS pubkey
-                  y2s_3             = y2s_3,            # PS pubkey
-                  y2s_4             = y2s_4,            # PS pubkey
-                  x2                = x2,               # PS pubkey
-                  close_flag        = close_flag,
+                    # elapsed between calling custClose and custClaim, and between calling expiry and merchClaim.
+                  delayExpiry       = sp.timestamp(0),  # if the delay is triggered, delayExpiry 
+                    #records when the delay is due to expire.
+                  g2                = g2,               # Pointcheval Sanders pubkey
+                  y2s_0             = y2s_0,            # Pointcheval Sanders pubkey
+                  y2s_1             = y2s_1,            # Pointcheval Sanders pubkey
+                  y2s_2             = y2s_2,            # Pointcheval Sanders pubkey
+                  y2s_3             = y2s_3,            # Pointcheval Sanders pubkey
+                  y2s_4             = y2s_4,            # Pointcheval Sanders pubkey
+                  x2                = x2,               # Pointcheval Sanders pubkey
+                  close_flag        = close_flag,       # A fixed scalar included in the message of
```

Comment:
Are we passing `close_flag` in as an argument for future customization of this field? Or is there another reason?

Reply:
We have it listed as a 'global default' in the spec, which I'd take to mean the merchant could customize it. Do we want it to be customizable?